lib/active_ldap/ldif.rb

```diff
@@ -138,7 +138,7 @@ def read_base64_value
       end
 
       def read_external_file
-        uri_string = @scanner.scan(URI::ABS_URI)
+        uri_string = @scanner.scan(URI::ABS_URI).chomp
```

Comment:
Method `uri_is_missing` was missing and untested. I've added test for missing or invalid URI